app/src/main/java/com/example/farmeraid/home/HomeViewModel.kt

```diff
@@ -59,4 +59,9 @@ class HomeViewModel @Inject constructor(
             message = "Navigates to Add Quota"
         )
     }
+    fun navigateToAddProduce() {
+        snackbarDelegate.showSnackbar(
+            message = "Navigates to Add Produce"
```

Comment:
Nit: Should it be navigate to add produce?

Reply:
demo purpose